webclient/components/papers/correlate/design-2.vue

```diff
@@ -1,23 +1,8 @@
 <template>
   <div>
     <el-button round type="primary" size="mini" @click="handleClickOnSettingsIcon">Correlate</el-button>
-    <el-dialog title="Correlate" :visible.sync="dIsSettingsDialogVisible" width="100%" top="5vh">
-      <div slot="header" class="s-css-class-outer-most-card-header clearfix">
-        <div style="display: grid; grid-template-columns: 1fr 5fr">
-          <tags-input
-            element-id="tags"
-            v-model="selectedSeriesTags"
-            :existing-tags="availableSeriesTags"
-            :typeahead="true"
-            :typeahead-activation-threshold="0"
-            :typeahead-hide-discard="true"
-            :only-existing-tags="true"
-            @tag-added="mfTagAdded"
-            :placeholder="cfGetPlaceholder"
-          ></tags-input>
-        </div>
-      </div>
-      <highcharts :options="cfChartOptions"></highcharts>
+    <el-dialog title="Correlate" :visible.sync="dIsSettingsDialogVisible" width="80%" top="5vh">
+      <graph></graph>
```

Comment:
1. Add documentation - Why we change el-dialog width from 100% to 80%?
2. Improve name. It should be 'correlateGraphCt' instead of 'graph'